### DISCUSSION TITLE
Button press works even when you click white space between options

### DISCUSSION THREAD

**Discussion by auraangelica** [jspsych/jsPsych#1502] [Q&A]
Hey y'all: 

I'm running an experiment that gives subjects 3 button choices. I was just editing it when I realized that clicking close to, but  outside, of a button-box still gets counted as a click.  Anyone know what this is due to/how to fix it? 

I'm looking at the .css and can see parameters like `padding` and `margin` but I don't know if/which of those would be of relevance. 

Thank you!

**Comment by jodeleeuw** [comment]
Hi @auraangelica,

Can you post the parameters for the trial that produces this behavior? I'm having a hard time reproducing the problem.

**Reply by auraangelica** [reply]
` for (i = 0; i < Astim.length; i++) {
  timeline.push({
    type: 'html-button-response',
    stimulus: '<p style="color: Blue; font-size: 22px; font-weight: normal;"> Choose the verb that you think is the "odd one out".</p><br>',
    choices: Astim[i],
    prompt: '<br><br><br>trial number: ' + i, 
  });
} `

(Unsure if this is what you meant?) 
Thank you!
